globals.go

```diff
@@ -566,6 +580,10 @@ var (
 		Description: "Not able to view logs",
 		Value:       0,
 	}
+	DistroSettingsCreate = PermissionLevel{
```

Comment:
This should be a separate permission, rather than a new level on the existing permission